src/components/Send.jsx

```diff
@@ -299,15 +299,18 @@ export default function Send({ makerRunning, coinjoinInProcess }) {
 
             <rb.Fade in={!isCoinjoinOptionEnabled} mountOnEnter={true} unmountOnExit={true}>
               <div className="mb-4 p-3 border border-1 rounded">
-                🛈{' '}
```

Comment:
Remove on [Earn](https://github.com/joinmarket-webui/joinmarket-webui/blob/master/src/components/Earn.jsx#L244) page as well?

Reply:
Removed. What was that even?

Reply:
It's an info sign. [Unicode Character 'CIRCLED INFORMATION SOURCE' (U+1F6C8)](https://www.fileformat.info/info/unicode/char/1f6c8/index.htm)